### DISCUSSION TITLE
Got javafx-libs for mac

### DISCUSSION THREAD

**Discussion by erik-forsgren** [AlmasB/FXGL#1129] [Q&A] [answered]
Hello, I'm not sure if I should report this as a bug instead, but when pulling FXGL as a gradle dependency (Gradle 7.3.1), I kept getting errors concerning prism and more specifically es2. 

So I started debugging my javaFX setup (running arch linux so I assumed the fault was with my env), however I couldn't find any obvious errors and openGL-stuff seemed to be working fine for other applications and so on. So I started checking which libs where actually pulled by my gradle application, and noticed that I had these to bad boys in my list of libraries **Gradle: org.openjfx:javafx-graphics:mac:16** and **Gradle: org.openjfx:javafx-base:mac:16**

There's probably some nice solution to this, but I resorted to specifying my own runtimeOnly dependencies for these to libs, and using the linux classifier instead, and voila; got my HelloWorld to run.

I couldn't find any other reference to this problem so just figured I'd post it here, please feel free to explain to me if I'm just stupid or if this has been discussed someplace already
Regards 
Erik

**Comment by AlmasB** [comment, marked as answer]
@erik-forsgren Hi, thanks for this. Have a look at the [template](https://github.com/AlmasB/FXGL-MavenGradle) for gradle and maven and see if that's more suitable. As far as I know gradle (unlike maven) doesn't pull the transitive dependencies, hence the need to explicitly specify them. Though I don't use gradle often to know this for a fact.

**Reply by erik-forsgren** [reply]
Ah, I see! Thank you, and keep up the good work!
